test/fixtures/mson-resource-unresolved-reference.apib

```diff
@@ -0,0 +1,11 @@
+# Polls [/]
+
+FIXME: what should be result of expansion for nonexistent structure?
```

Comment:
Drafter shouldn't even think of trying to expand this example since snowcrash gives an error.

Once the parser (snowcrash/drafter) sees an error, it stops doing everything and returns an error to the user. Why are we testing the expansion here?